packages/framework/src/Model/Order/OrderData.php

```diff
@@ -196,7 +196,7 @@ public function getNewItemsWithoutTransportAndPayment()
     {
         $newItemsWithoutTransportAndPayment = [];
         foreach ($this->itemsWithoutTransportAndPayment as $index => $item) {
-            if (strpos($index, self::NEW_ITEM_PREFIX) === 0) {
+            if (strpos((string)$index, self::NEW_ITEM_PREFIX) === 0) {
```

Comment:
When we make changes here, I would rather just use

`str_starts_with((string)$index, self::NEW_ITEM_PREFIX)`

Reply:
That is out of scope change actually, but I did as you wished